src/mqtt/fbws.js

```diff
@@ -0,0 +1,93 @@
+//From mqtt.js
```

Comment:
Is this copied from somewhere? It's ok, but can you put a more full URL or something so that we can find it again later?

Reply:
Sure. I will add an url to this.

Reply:
Sure. I will add an url to this.